codeexamples/eventtracking.js

```diff
@@ -1,57 +1,99 @@
-function createEventsExample(client, productId) {
-    console.log("create-events:");
-    
+// Multi Event Tracking Example
+function createMultiEventsExample(client, productIds) {
+    console.log("create-multi-events:");
+    console.log(productIds);
+
     // CODE SAMPLE create-events BEGIN
     // click event (can be called on the product page)
-    var clickedEntity = {type: "Product", id: productId};
+    var clickedEntities = productIds.map(function (value, index, array) {
+        return { type: "click", entity: { type: "Product", id: value } };
+    });
+
+    client.createEvents(clickedEntities, response => {
+        console.log("click events response", response);
+    });
+
+    // addtocart event (call this when a customer adds a product to cart)
+    var addToCartEntities = productIds.map(function (value, index, array) {
+        return { type: "addtocart", entity: { type: "Product", id: value } };
+    });
+    client.createEvents(addToCartEntities, response => {
+        console.log("add to cart response", response);
+    });
+
+    // purchase events (can be called when an order is processed, or on the "thank you" page)  
+    // orderId is Optional but recommended
+    // quantity is optional
+    // revenue is optional
+    var purchasedEntities = productIds.map(function (value, index, array) {
+        return { type: "purchase", entity: { type: "Product", id: value }, orderId: "13t09j1g", quantity: 5, revenue: 249.0 };
+    });
+
+    //createEvent also works with promises
+    var purchasePromise = client.createEvents(purchasedEntities).then(response => {
+        console.log("purchase response", response);
+    });
+    // CODE SAMPLE END
+    purchasePromise.then((r) => console.log("create-multi-events (end)"))
+
+    // TODO : Add Custom Data Examples
+};
+
+// Single Event Tracking Example
+function createSingleEventExample(client, productId) {
+    console.log("create-sigle-event:");
+
+    // CODE SAMPLE create-event BEGIN
```

Comment:
These CODE SAMPLE segments are for our public documentation to fetch code from here. I'm thinking about whether we want to change those.

Reply:
I see - ok - I guess I'll leave that with you - let me know if you want to leave the CODE SAMPLE comments in there or not

Reply:
I think it's best to leave the CODE SAMPLE comments in this file as they were, and just not have any CODE SAMPLE sections for the new functions. We should probably sort it out and have separate examples on docs.loop54.com, but then we need to make the same change in the other 3 libraries and I think that's out of scope for now.